### DISCUSSION TITLE
Главный чат

### DISCUSSION THREAD

**Discussion by scratch-craft-2** [scratch-craft-2/ya-games-dash#1] [General]
Чат

**Comment by gervaty** [comment]
А почему это расширение тута, оно же в официальной галереи экстеншенов турбика есть

**Reply by scratch-craft-2** [reply]
... Идиот, во первых мое дело куда пихать мое расширение во вторых я не хочу копаться с добавлением своего расширения, я уже модалы хотел добавить, в пятых отвали че ты вообще ко мне пристал идиот

**Reply by scratch-craft-2** [reply]
Ну че ты ко мне вообще придрлблася дебил сначала проверяй а потом думай во вторых должен быть репозиторий с исходным кодам и вообще иди нафиг че пристал ко мне идиот

**Reply by gervaty** [reply]
1. откуда столько агрессии?
2. пару месяцев назад это расширение реально было в галереи
